src/app/content/studyGuides/hooks/receiveBook.ts

```diff
@@ -0,0 +1,22 @@
+import { GetHighlightsSummarySourceTypeEnum } from '@openstax/highlighter/dist/api';
+import { ActionHookBody, AppServices, MiddlewareAPI } from '../../../types';
+import { actionHook } from '../../../utils';
+import { receiveBook } from '../../actions';
+import { receiveStudyGuides } from '../actions';
+
+export const hookBody: ActionHookBody<typeof receiveBook> = (
```

Comment:
better to hook into locationChange and check the book state. in production receiveBook will be called at build time, not on page load, which would mean the summary could get very stale between releases

Reply:
Hmm, but `locationChange` is dispatched before `book`

Reply:
Ok, I see, I have to hook it to the content locationChange

Reply:
yea if you call it from there it makes sure the book exists